pkg/plugin/README.md

```diff
@@ -0,0 +1,26 @@
+# Aethers Plugin Systems
+
+We have based our plugin system off of [hashicorps go-plugins](https://github.com/hashicorp/go-plugin/tree/main). This was mainly so we could detach the need for matching dependencies. We use gRPC for communication
+
+## Example Plugin
+
+There is an example plugin in the [example directory](./example/example.go) with comments on the moving parts. 
+
+## Install a Plugin
+
+// TODO //
```

Comment:
maybe create issues for these TODOs to track them

Reply:
https://github.com/re-cinq/aether/issues/77